gflownet/utils/batch.py

```diff
@@ -0,0 +1,179 @@
+from collections import defaultdict
+from copy import deepcopy
+
+import numpy as np
+import torch
+
+from gflownet.utils.common import tbool, tfloat, tint, tlong
+
+
+class Batch:
+    """
+    Important note: one env should correspond to only one trajectory, all env_id should be unique
+    """
+
+    def __init__(self, loss, device, float):
```

Comment:
Is it possible to use a default value for the float here? Or is this actually problem-dependent? Partially so that it gives an example of what should it be.

Reply:
I have no idea 😅 that's something I follow from the GFlowNetAgent class, it sets all the floats in tensors to self.float. And Batch class just takes this self.float from the agent and sets it to the tensors created inside its methods